scalding-core/src/main/scala/com/twitter/scalding/serialization/CascadingBinaryComparator.scala

```diff
@@ -94,7 +94,7 @@ object CascadingBinaryComparator {
     else {
       val badSteps = missing.size
       val msg = missing.zipWithIndex.map { case (msg, idx) => s"<step$idx>$msg</step$idx>" }.mkString
-      error(s"There are $badSteps missing OrderedSerializations: $msg")
+      sys.error(s"There are $badSteps missing OrderedSerializations: $msg")
```

Comment:
ahhh This was a bug!!!! Our error was out of scope here so it was hitting `Predef.error` which is not what we intended!  :)